### PR TITLE
101 hdinsight linux with existing linked storage account fix

### PR DESCRIPTION
**Pull request by JavidSalmanov**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-hdinsight-linux-with-existing-linked-storage-account/prereqs/prereq.azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        "name": "[parameters('storageAccountType')]"
+      },
+      "kind": "StorageV2",
+      "properties": {}
+    }
+  ],
+  "outputs": {
+    "linkedSAName": {
+      "type": "string",
+      "value": "[replace(replace(concat(reference(concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName')), '2020-08-01-preview').primaryEndpoints.blob),'https:',''),'/','')]"
```
Comment (bmoore-msft) [inline 101-hdinsight-linux-with-existing-linked-storage-account/prereqs/prereq.azuredeploy.json:45]: remove first concat (it's not used) replace second concat with resourceId()

File: 101-hdinsight-linux-with-existing-linked-storage-account/README.md [hunk 2]
```diff
@@ -6,6 +6,9 @@
 ![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-hdinsight-linux-with-existing-linked-storage-account/FairfaxLastTestDate.svg)
 ![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-hdinsight-linux-with-existing-linked-storage-account/FairfaxDeployment.svg)
 
+![sanitize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)
```
Comment (bmoore-msft) [inline 101-hdinsight-linux-with-existing-linked-storage-account/README.md:9]: fix buttons (See other samples if needed)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
also check pipeline results for other changes

File: 101-hdinsight-linux-with-existing-linked-storage-account/README.md (hunk 2 above)
Comment (JavidSalmanov) [reply 101-hdinsight-linux-with-existing-linked-storage-account/README.md]: Could you check again?

**Comment by JavidSalmanov** [comment]
> also check pipeline results for other changes

Pipelines result is okay. I changed the Readme file based on other samples.

**Comment by JavidSalmanov** [comment]
@bmoore-msft I really don't understand why the readme check fails. I checked the readme files and followed the same scenario. Could you give a hint of what is wrong with the readme?

**Comment by bmoore-msft** [comment]
@JavidSalmanov - I fixed the readme issues - there are a few best practice violations that still need to be addressed.

**Comment by JavidSalmanov** [comment]
> @JavidSalmanov - I fixed the readme issues - there are a few best practice violations that still need to be addressed.

@bmoore-msft issue has been fixed

**Pull request merged**